lsp/client-extension/default.nix

```diff
@@ -1,8 +1,11 @@
 # This file has been generated by node2nix 1.11.1. Do not edit!
 
-{pkgs ? import <nixpkgs> {
+{ pkgs ? import <nixpkgs> {
```

Comment:
This comment applies to this file and the 2 other generated by `node2nix`: please don't format them. It's annoying because every time someone regenerates them, it leads to huge diffs, and forces to re-format those files.

See https://github.com/tweag/nickel/blob/master/flake.nix#L134-L138